src/components/Fund/FundDetails/FundDetailsContainer.js

```diff
@@ -184,14 +188,16 @@ const FundDetailsContainer = ({
   );
 
   const addBudgetButton = useCallback((status, count) => {
-    return !count && (
-      <Button
-        data-test-add-budget-button
-        onClick={() => setBudgetStatusModal(status)}
-      >
-        <FormattedMessage id="ui-finance.budget.button.new" />
-      </Button>
-    );
+    return !count
```

Comment:
it's a fix of warning in console, when a boolean goes further and isn't expected